src/content/release-notes.mdx

```diff
@@ -5,6 +5,33 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.38.0
+
+[Date TBD]
+
+This version is compatible with Kubernetes versions 1.31 to 1.33 \
+Okteto Chart release 1.38 is designed to work with [Okteto CLI 3.13.x](https://github.com/okteto/okteto/releases/tag/3.13.0)
+
+### New Features {#new-features-1.38}
+
+- **Enhanced Preview Environments UI**: Dramatically improved loading performance, reducing load times by up to 15x for customers with large numbers of Preview Environments. Added comprehensive filtering options (search, repository, status, owner, and time-based filters) to help teams quickly find the previews they need. Also includes UI refinements such as a personal scope indicator icon, reorganized columns, and abbreviated repository names for better readability <!-- 9761 -->
+- Added read-only mode for Known Hosts configuration, providing enhanced control over SSH host key management <!-- 9726 -->
+
+### Improvements {#improvements-1.38}
+
+- Upgraded to BuildKit v0.25.1 for improved build performance and stability <!-- 9725, 9731 -->
+- Updated Okteto AI Agent base image to Debian Trixie with Go 1.24.6 and Node.js 24 <!-- 9729 -->
+- Okteto AI: Added support for Claude Sonnet 4.5, the latest AI model optimized for software development tasks <!-- 9745 -->
+- Improved Preview Environments filter ordering to match column organization <!-- 9773 -->
+- [Okteto CLI 3.13.0](https://github.com/okteto/okteto/releases/tag/3.13.0-beta.1): Enhanced namespace destruction logic and error handling during resource cleanup
+
+### Bug Fixes {#bug-fixes-1.38}
+
+- Fixed Known Hosts configuration for main repository cloning <!-- 9738 -->
+- Fixed Okteto AI Agent list display issues <!-- 9749 -->
```

Comment:
I think this can be omitted, this wasn't go live as it was a regression during the sprint, but we detected it before going live.